azure/template.json

```
@@ -110,16 +110,23 @@
       }
     },
     "settingsAuthenticationSecret": {
-        "type": "string",
-        "metadata": {
-            "description": "Auth secret for manage back-end."
-        }
+      "type": "string",
+      "metadata": {
+        "description": "Auth secret for manage back-end."
+      }
     },
     "settingsManageApiSecret": {
-        "type": "string",
-        "metadata": {
-            "description": "The secret shared between manage courses backend and manage courses API to authorize requests."
-        }
+      "type": "string",
+      "metadata": {
+        "description": "The secret shared between manage courses backend and manage courses API to authorize requests."
+      }
+    },
+    "databaseServiceName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The abbriviation of the database service name to be used when naming a resource."
```

```suggestion
        "description": "The abbreviation of the database service name to be used when naming a resource."
```

typo for `abbreviation`

---

You should spot the earlier one as well

---

is this actual `databaseServiceName` not 

```json
"databaseResourceNamePrefix": "[toLower(concat('bat-', parameters('resourceEnvironmentName'),'-', parameters('databaseServiceName')))]"
```

seems a touch misleading
```json
"databaseServerName": "[concat(variables('databaseResourceNamePrefix'), '-psql')]",
```